.github/workflows/ci.yml

```diff
@@ -176,6 +176,5 @@ jobs:
         # https://github.com/RasaHQ/rasa-demo/blob/main/.github/workflows/continuous_integration.yml#L199-L207
         test_core_args: >-
           --stories tests/test_conversations.yml
-          --fail-on-prediction-errors
```

Comment:
Why did you remove it?

Reply:
The `--fail-on-prediction-errors` flag is used in the rasa-demo so I included it also in this action. As I working on patches, I found that the presence of prediction errors is a random thing. I often need to re-run the workflow till no more prediction errors. And yes, we can keep the flag if this is part of the feature we want to test.

Reply:
What do you think?